tools/check_format.sh

```diff
@@ -15,7 +15,6 @@ TO_CHECK="${2:-$FULL_CHECK}"
 bazel run @envoy//tools/code_format:check_format.py -- \
   --skip_envoy_build_rule_check  --namespace_check Nighthawk \
   --build_fixer_check_excluded_paths=$TO_CHECK \
-  --include_dir_order envoy,nighthawk,external/source/envoy,external,api,common,source,exe,server,client,distributor,sink,grpcpp,request_source,test_common,test \
```

Comment:
Can we say something about the removal of this argument (its migration to the yaml) in the description of the PR for posterity?

Reply:
This was in the PR, but I had accidentally called it header_dir_order. Fixed that typo